lib/docker/compose/session.rb

```diff
@@ -175,28 +175,20 @@ def  build(*services, force_rm:false, no_cache:false, pull:false)
     # @return [String] output of the command
     # @raise [Error] if command fails
     def run!(*args)
-      project_opts = {}
-      case @file
+      file_args = case @file
+        when 'docker-compose.yml'
+          []
```

Comment:
this case would have been handled by the lower to string, does it make sense to explicitly handle it?